### ISSUE TITLE
Accuracy on Nuscene dataset

### ISSUE THREAD

**Issue by chowkamlee81** [kosuke55/train_baiducnn#6]
Would like to know accuracy on trained with Nuscene dataset? Is it trained for sweeps=1 or sweeps=10. Kindly help

**Comment by kosuke55** [comment]
@chowkamlee81 
Thanks for the comment. I haven't done the nuScenes detection task yet, so I can't answer the specific accuracy. To get a rough idea of the accuracy, try [this prediction code](https://github.com/tier4/AutowareArchitectureProposal/tree/master/src/perception/object_recognition/detection/lidar_apollo_instance_segmentation) with the trained model.
The number of sweeps can be adjusted here. The default value is 10.

https://github.com/kosuke55/train_baiducnn/blob/2c06a6bd11c7e12dcfc66ba7cf31be3bcc232031/scripts/create_dataset/create_dataset_from_nusc.py#L61-L62

**Comment by chowkamlee81** [comment]
@kosuke55 , did you got enough time to train successfully models with decent accuracy on Nuscenes dataset. Kindly help

**Comment by kosuke55** [comment]
Yes, it may take a day or so. Look at the graph and see if your losses stop dropping.
